common/lib/xmodule/xmodule/js/spec/problem/edit_spec.coffee

```diff
@@ -363,22 +360,20 @@ describe 'MarkdownEditingDescriptor', ->
         Multiple Choice also allows students to select from a variety of pre-written responses, although the format makes it easier for students to read very long response options. Optionresponse also differs slightly because students are more likely to think of an answer and then search for it rather than relying purely on recognition to answer the question.
         [Explanation]
         """)
-      expect(data).toXMLEqual("""<problem>
-        <optionresponse>
-          <p>OptionResponse gives a limited set of options for students to respond with, and presents those options in a format that encourages them to search for a specific answer rather than being immediately presented with options from which to recognize the correct answer.</p>
-        <p>The answer options and the identification of the correct answer is defined in the <b>optioninput</b> tag.</p>
-        <p>Translation between Option Response and __________ is extremely straightforward:</p>
-        <optioninput options="('Multiple Choice','String Response','Numerical Response','External Response','Image Response')" correct="Multiple Choice"/>
-        </optionresponse>
-
-        <solution>
-        <div class="detailed-solution">
-        <p>Explanation</p>
-
-        <p>Multiple Choice also allows students to select from a variety of pre-written responses, although the format makes it easier for students to read very long response options. Optionresponse also differs slightly because students are more likely to think of an answer and then search for it rather than relying purely on recognition to answer the question.</p>
-
-        </div>
-        </solution>
+      expect(data).toXMLEqual("""
+        <problem>
+            <optionresponse>
+                <p>OptionResponse gives a limited set of options for students to respond with, and presents those options in a format that encourages them to search for a specific answer rather than being immediately presented with options from which to recognize the correct answer.</p>
+                <p>The answer options and the identification of the correct answer is defined in the <b>optioninput</b> tag.</p>
+                <p>Translation between Option Response and __________ is extremely straightforward:</p>
+                <optioninput options="('Multiple Choice','String Response','Numerical Response','External Response','Image Response')" correct="Multiple Choice"/>
+                <solution>
+                    <div class="detailed-solution">
+                    <p>Explanation</p>
+                    <p>Multiple Choice also allows students to select from a variety of pre-written responses, although the format makes it easier for students to read very long response options. Optionresponse also differs slightly because students are more likely to think of an answer and then search for it rather than relying purely on recognition to answer the question.</p>
+                    </div>
+                </solution>
```

Comment:
NIT: Can you add indentation to all the solution tag's content. It's almost in every test in this file.